app/src/main/java/io/appium/uiautomator2/model/Session.java

```diff
@@ -33,15 +34,17 @@ public String getSessionId() {
 
     public static boolean shouldUseCompactResponses() {
         boolean shouldUseCompactResponses = true;
-        if (Session.capabilities.containsKey(CAP_SHOULD_USE_COMPACT_RESPONSES)) {
-            shouldUseCompactResponses = BooleanUtils.toBoolean(Session.capabilities.get(CAP_SHOULD_USE_COMPACT_RESPONSES).toString());
+        if (Session.capabilities.containsKey(SHOULD_USE_COMPACT_RESPONSES.toString())) {
+            shouldUseCompactResponses = BooleanUtils.toBoolean(
+                    Session.capabilities.get(SHOULD_USE_COMPACT_RESPONSES.toString()).toString());
         }
         return shouldUseCompactResponses;
     }
 
     public static String[] getElementResponseAttributes() {
-        if (Session.capabilities.containsKey(CAP_ELEMENT_RESPONSE_ATTRIBUTES)) {
-            return Session.capabilities.get(CAP_ELEMENT_RESPONSE_ATTRIBUTES).toString().split(",");
+        if (Session.capabilities.containsKey(ELEMENT_RESPONSE_ATTRIBUTES.toString())) {
+            return Session.capabilities.get(ELEMENT_RESPONSE_ATTRIBUTES.toString()).toString().
+                    split(",");
```

Comment:
in java the dot separator is usually the first character if wrapped to a new string:

```
.toString()
.split(",")
```